package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "posterior",
+  "version": "1.0.0",
+  "description": "Backend for the Alcoding Web portal",
+  "main": "app.js",
+  "scripts": {
+    "start:dev": "nodemon",
+    "build": "rimraf ./build && tsc",
+    "start": "npm run build && node build/app.js",
+    "lint": "eslint . --ext .ts",
+    "lint-and-fix": "eslint . --ext .ts --fix",
+    "prettier-format": "prettier --config .prettierrc 'src/**/*.ts' --write",
+    "prettier-watch": "onchange 'src/**/*.ts' -- prettier --write {{changed}}"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run prettier-format && npm run lint"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pes-alcoding-club/posterior.git"
+  },
+  "keywords": [
+    "alcoding",
+    "backend"
+  ],
+  "author": "pes-alcoding-club",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/temanisparsh/posterior/issues"
+  },
+  "homepage": "https://github.com/temanisparsh/posterior#readme",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "errorhandler": "^1.5.1",
+    "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^5.9.22",
+    "multer": "^1.4.2",
```

Comment:
What do we need multer for?

Reply:
We don't need multer for anything right now(Maybe for profile page) , but in future we will need it for file upload in general.

Reply:
Let's remove it for now. According to bundlephobia it is one of the heaviest package in our project. When we need to add the upload functionality, we'll weigh in other alternatives and choose.

Reply:
Alright, will remove it

Reply:
See issue #7